src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+
+interface Props {
+  activeTodosLength: number;
+  completedTodosLength: number;
+  filterStatus: string;
+  changeTodosStatus: (status: string) => void;
+  removeAllTodos: () => void;
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  activeTodosLength,
+  completedTodosLength,
+  filterStatus,
+  changeTodosStatus = () => {},
+  removeAllTodos = () => {},
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodosLength} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filterStatus === Status.all,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => changeTodosStatus(Status.all)}
+        >
+          {Status.all}
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: filterStatus === Status.active,
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => changeTodosStatus(Status.active)}
+        >
+          {Status.active}
+        </a>
+
+        <a
+          href="#/completed"
+          className={cn('filter__link', {
+            selected: filterStatus === Status.completed,
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => changeTodosStatus(Status.completed)}
+        >
+          {Status.completed}
+        </a>
+      </nav>
+
+      {completedTodosLength > 0 && (
```

Comment:
do not render clear completed button conditionally.
it should be always rendered but in case no comleted todos should has disabled attribute.
it can fix some tests.
and will place filter buttons in proper position
![Screenshot from 2023-10-24 14-51-59](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/4ca7cccf-58d1-4610-8f9c-c13ad3684a43)